buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,382 @@
+# Mapping between old and new commondata for oldcommondata_porter
+#
+# <old_commondata_name>:
+#   new_name: <new_name>
+#   reference_arxiv: <http://etc>
+#   reference_hepdata: <http://etc>
+#
+
+# HERA
+HERACOMBCCEM:
+  new_name: HERA_CC_318GEV_EM-SIGMARED
```

Comment:
Is there a difference between data and theory concerning naming? i.e. why is there a `-` instead of a `_` - compare https://github.com/NNPDF/pinecards/tree/master/HERA_CC_318GEV_EM_SIGMARED

Reply:
Yes. We decided at some point that the last `_` defined what the observable was and everything before was the set of data.
So I went for `HERA_CC_318GEV` for the set of data and `EM-SIGMARED` for the observable.

Reply:
fine - but then I would suggest to slightly swap to `SIGMARED_EM` since we are clearly going "general->detail" ... this would make it also more similar to its sister dataset `SIGMARED_EP`

Reply:
That's a very good suggestion. Actually, given that all of them are `sigmared` we might want to even drop it?

Reply:
> Actually, given that all of them are `sigmared` we might want to even drop it?

SLAC and BCDMS are actually F2

Reply:
All the HERA I meant. But it's true, it doesn't hurt to keep it.

Also I saw what you meant now with the journal, since it came from a different source I missed that they were coming from the same reference!

I'll push the mapping with your suggestions. I still haven't changed the energy, but I like the `NOTFIXED` suggestion.

Reply:
What I'd propose goes along Felix's suggestion, ie doing `_EM-SIGMARED` instead.

Reply:
Except I just [argued](https://github.com/NNPDF/nnpdf/pull/1931#discussion_r1481482698) for the opposite